src/commcare_cloud/ansible/roles/postgresql_base/templates/postgresql.conf.j2

```diff
@@ -130,7 +130,13 @@ track_activity_query_size = 2048
 # AUTOVACUUM PARAMETERS
 
 autovacuum = on
-
+{% if not is_pg_standby and  inventory_hostname in groups['synclog_dbs'] or inventory_hostname in groups['shard_dbs'] %}
```

Comment:
why do we want to restrict this to `not is_pg_standby`?

Also if we want to customize it based on VM type we should use variables that get changed in the inventory files.

Reply:
> why do we want to restrict this to `not is_pg_standby`?
>
Vacuum and autovacuum replicate like any other write operation.Running vacuum or autovacuum on the slave doesn't do anything and is not necessary.

> Also if we want to customize it based on VM type we should use variables that get changed in the inventory files. l
> 
Like adding another Host Level Vars?

Reply:
> Vacuum and autovacuum replicate like any other write operation.Running vacuum or autovacuum on the slave doesn't do anything and is not necessary.

Sure but if we failover to a standby then we'd want this right? I don't think the autovacuum process runs when the DB is in recovery mode.

> Like adding another Host Level Vars?

Yes, or group level

Reply:
Updated